app/components/dashboard/DcChart.vue

```
@@ -621,36 +635,79 @@
+              </div>
+            </template>
+            <template v-if="item.isDcSunburstChart">
+              <DcSunburstChart
+                v-if="mm.isLoadAllData"
+                :ndx="mm.ndx"
+                :chartId="`chart_ex_${i}`"
+                chartGroup="chartGroup"
+                v-model:filters="mm.chartEx[i].filters"
+                :innerRadius="mm.opt.chartEx[i]?.innerRadius"
+                :showLabels="true"
+                :legend="mm.opt.chartEx[i]?.isLegend ? mm.opt.chartEx[i]?.dcSunburstChart.legend: {isShow: false}"
```

ESLint

[error] A space is required after '{'.

(vue/object-curly-spacing)